packages/admin-panel/src/VizBuilderApp/components/PreviewSection.js

```diff
@@ -100,7 +112,7 @@ const getColumns = data => {
   const columns = columnKeys.map(columnKey => {
     return {
       Header: columnKey,
-      accessor: row => row[columnKey],
+      accessor: row => convertValueToPrimitive(row[columnKey]),
```

Comment:
Otherwise it will error, I took a look at the react table docs (which is where this is eventually rendered) and they say that accessor can only return a primitive value. I hope this is ok to slip in, as it will only ever matter when `val` is an object or a function, where previously it would give an unhelpful error (sorry I forget what it was)